components/Header/Header.js

```diff
@@ -15,6 +15,53 @@ import updateTrap from '../../actions/loginModal/updateTrap';
 let MediaQuery = require ('react-responsive');
 class Header extends React.Component {
 
+    constructor(props){
```

Comment:
I would prefer to revert these changes as it was way less code before and thus less to read/search for if someone needs to change something.